src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,39 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonRow } from './PersonRow';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable:React.FC<Props> = ({ people }) => {
+  const { personSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
Here you can cheat a little and make the code more readable and scalable if we take out the names of the columns in a separate array and `map` it as we do with `people`.